test/hga/HGA_regression.ipynb

```diff
@@ -0,0 +1,2456 @@
+    "                Environment.UAT: 'G1244144598-EEDTEST',\n",
+    "                Environment.SIT: 'G1244144598-EEDTEST'}\n",
+    "\n",
+    "    avnir_directory_three = 'avnir_test_three'\n",
+    "    \n",
+    "    avnir_request_three = Request(collection=environment_information['avnir_collection'],\n",
+    "                                  spatial=BBox(179.75, 66.5, -179.85, 66.75),\n",
+    "                                  variables=['Band1', 'Band2'], format='image/tiff',\n",
+    "                                  granule_id=granules[environment_information['env']])\n",
+    "\n",
+    "    avnir_output_three = make_request_and_download_result(harmony_client, avnir_request_three,\n",
+    "                                                          avnir_directory_three)\n",
+    "\n",
+    "    avnir_variable_base_three = 'ALAV2A239652240-OORIRFU_subsetted'\n",
+    "\n",
+    "    avnir_variables_three = {f'{avnir_variable_base_three}__Band1__colored_band_1',\n",
+    "                             f'{avnir_variable_base_three}__Band1__colored_band_2',\n",
+    "                             f'{avnir_variable_base_three}__Band1__colored_band_3',\n",
+    "                             f'{avnir_variable_base_three}__Band1__colored_band_4',\n",
+    "                             f'{avnir_variable_base_three}__Band2__colored_band_1',\n",
+    "                             f'{avnir_variable_base_three}__Band2__colored_band_2',\n",
+    "                             f'{avnir_variable_base_three}__Band2__colored_band_3',\n",
+    "                             f'{avnir_variable_base_three}__Band2__colored_band_4'}\n",
+    "\n",
+    "    expected_avnir_results_three = {'authority': 'EPSG',\n",
+    "                                    'cs': 'Projected',\n",
+    "                                    'gcs': 'WGS 84',\n",
+    "                                    'gcs_epsg': '4326',\n",
+    "                                    'n_bands': 8,\n",
+    "                                    'proj_cs':'WGS 84 / UTM zone 1N',\n",
+    "                                    'proj_epsg': '32601',\n",
+    "                                    'spatial_extent': [66.44, 66.81, 179.85, -179.95],\n",
+    "                                    'variables': avnir_variables_three,\n",
+    "                                    'width': 2694,\n",
+    "                                    'height': 3250}\n",
+    "    \n",
+    "    check_request_output(avnir_output_three[0], expected_avnir_results_three)\n",
+    "    clean_test_artefacts(avnir_directory_three)\n",
+    "else:\n",
+    "    print('Skipping test: HGA regression tests not configured for this environment.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97518d21",
+   "metadata": {},
+   "source": [
+    "### AVNIR test case 4:\n",
+    "\n",
+    "This test case combines a bounding box spatial subset in the Northern Hemisphere with a wariable subset requesting only band 4.\n",
```

Comment:
```suggestion
    "This test case combines a bounding box spatial subset in the Northern Hemisphere with a variable subset requesting only band 4.\n",
```

Reply:
"Where are the nuclear wessels?" (Thanks - I'l update this now)